msal/src/androidTest/java/com/microsoft/identity/client/PublicClientConfigurationTest.java

```diff
@@ -0,0 +1,64 @@
+package com.microsoft.identity.client;
+
+import android.support.test.runner.AndroidJUnit4;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@RunWith(AndroidJUnit4.class)
+public class PublicClientConfigurationTest {
+
+    /**
+     * Verifies that a minimum configuration containing only:
+     * client_id
+     * redirect_uri
```

Comment:
Are we checking only that these values were not null? Do we need to check a specific schema? (GUID, prefix, anything more than != null?)

Reply:
Not null is sufficient.  Just verifying that we have a value is sufficient for now.  Their may be more specific validation required for redirect_uri at some point.